integration/__init__.py

```diff
@@ -0,0 +1,23 @@
+from marionette import Marionette, MarionetteTestCase
+from firefox_puppeteer.testcases import BaseFirefoxTestCase
+
+
+class FirefoxTestCase(BaseFirefoxTestCase, MarionetteTestCase):
+
+    # HACK: Replicate setUp from superclass, but just without the navigation to
+    # browser.newtab.url
+    def setUp(self, *args, **kwargs):
```

Comment:
It would be good to set the element search timeout to a generous number inside this setUp method, so that you can call find_element without having to use as many explicit waits. I think this would do it: `self.marionette.set_search_timeout(10000)`.

I tried looking in the [BaseFirefoxTestCase](https://dxr.mozilla.org/mozilla-central/source/testing/puppeteer/firefox/firefox_puppeteer/testcases/base.py), and it doesn't seem to set the search timeout either. I could be missing it in another base class, or a config file, though.